services/api/src/index.ts

```diff
@@ -487,21 +604,31 @@ server.after(() => {
             response: { 200: Dto.checkLoginStatusResponse },
         },
         handler: async (request) => {
-            const { didWrite } = await verifyUCAN(db, request, {
-                expectedDeviceStatus: undefined,
-            });
-
-            const status = await authUtilService.getDeviceStatus(db, didWrite);
-            const loggedInStatus: DeviceLoginStatus = !status.isRegistered
-                ? "unknown"
-                : !status.isVerified
-                ? "unverified"
-                : !status.isLoggedIn
-                ? "logged_out"
-                : "logged_in";
-            return {
-                loggedInStatus: loggedInStatus,
-            };
+            const { deviceStatus } = await verifyUcanAndDeviceStatus(
+                db,
+                request,
+                {
+                    expectedDeviceStatus: undefined,
```

Comment:
By default the option assume "logged-in". The reason is, I don't want new developers ramping up knowledge on the codebase to inadvertently authorize an endpoint to everyone, thinking it's authenticated. As a result, you must manually put this option to undefined if you want to authorize it to anyone.